saleor/discount/migrations/tasks/saleor3_19.py

```diff
@@ -0,0 +1,27 @@
+from ....celeryconf import app
+from ... import DiscountType
+from ...models import CheckoutLineDiscount, OrderLineDiscount
+
+BATCH_SIZE = 250
```

Comment:
How did you decide on this batch size? Did you maybe measure the memory usage and time execution?

Reply:
The question still stands? How batch size was decided?

We need proper tests to measure memory usage and time execution. I can help with that.